src/main/daml/Daml/Finance/Holding/Fungible.daml

```diff
@@ -90,3 +96,27 @@ template Factory
       setObservers = setObserversImpl @Factory @Disclosure.I this None
       addObservers = addObserversImpl @Factory @Disclosure.I this None
       removeObservers = removeObserversImpl @Factory @Disclosure.I this None
+
+-- | Implementation of a factory for the factory template.
+-- It allows for creating a factory with its reference.
```

Comment:
`Reference` is a work-around we don't really explain in the docs, so I wonder whether it's confusing  to mention it here.

Reply:
I removed that line (in the other files as well)